assesment-week-1-coding-concepts.js

```diff
@@ -0,0 +1,58 @@
+// ASSESSMENT 1: Coding conceptual questions
+
+// Examine the following examples.
+
+// First, predict the outcome based on your understanding of the code.
+// Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
+
+// --------------------1) What will this log?
+//
+// var cohort = "Delta 2021"
+// console.log(cohort.length)
+//
+// a) Your answer: 10
+// b) Verify and explain:This will log the length of "delta 2021"
+
+
+// --------------------2) What will this log?
+//
+// var greeting = "Hello World!"
+// console.log(greeting[3])
+
+
+// a) Your answer:This Should log the location of the 3rd letter in the greeting which is "l" the second "l" in specific.
+
+// var greeting = "Hello World!"
+// console.log("greeting"[3]);
+
+// b) Verify and explain: Ok, It came back as "e" because we are logging "greeting" not "hellow world"
+
+
+// --------------------3) What will this log?
+
+// var languages = ["JavaScript", "Ruby", "Python", "C++"]
+// var index = 1
+// console.log(languages[index])
+
+// a) Your answer: Ruby
+// b) Verify and explain: because it is Counting JavaScript as the zeroth space and ruby would be index 1.
+
+
+// --------------------4) What will this log?
+
+// var weekendDays = ["saturday", "sunday"]
+// console.log(weekendDays.toUpperCase())
+
+// a) Your answer: It should show "SATURDAY" AND "SUNDAY"
+// b) Verify and explain: It is showing that .toUpperCase() is not a function when I run it. Either the upper case comand is not a recognised function in JavaScript or 'saturday' and 'sunday' are still in string form.
```

Comment:
Close, the strings are currently encapsulated by an array. So to access those values we'll have to call on those indexes ourselves or use some kind of iteration to access them and run .toUpperCase() on them.

Reply:
To get into the plastic baggy 😎
It's slowly sinking in!